src/components/Modals/SemesterMenu.vue

```diff
@@ -18,6 +18,11 @@ export default {
       this.$emit('open-delete-semester-modal');
       const semesterMenu = document.getElementById('semesterMenu');
       semesterMenu.style.display = 'none';
+    },
+    openEditSemesterModal() {
```

Comment:
This just does the same thing as a deleting semester modal right? Is this a WIP?